mozilla-release/browser/components/preferences/in-content/privacy.xul

```diff
@@ -652,9 +658,11 @@
   <separator flex="1"/>
 
   <hbox align="start" id="autoplayMediaCheckboxWrapper">
```

Comment:
@tsl143 align="start" means to align elements vertically starting from left top corner of hbox.
Could you please change it for "center" since autoplayMediaCheckbox and autoplayMediaPolicyButton are not centered vertically relatively each other now?

Reply:
This again comes from Firefox and maybe not in scope of this PR?

Reply:
TODO please. Otherwise we will forget about that.

Reply:
Instead lets create a ticket?

Reply:
Sure. Let's do like that.